bims/scripts/import_gbif_occurrences.py

```diff
@@ -136,6 +136,9 @@ def import_gbif_occurrences(
 
 
     for result in json_result['results']:
+        # Prevent pulling FBIS data back down from GBIF
+        if 'projectId' in result and result['projectId'].lower() == 'fbis':
```

Comment:
![image](https://user-images.githubusercontent.com/1979569/236387055-46a68115-c257-43fb-8231-eaffa908279a.png)

When retrieving data from GBIF, if the data originates from FBIS, it will have `projectId: 'FBIS'`. In this case, the system will simply skip over that data and proceed to the next available data point.

Reply:
cc @jeremyshelton